keyboards/gmmk/pro/readme.md

```diff
@@ -17,7 +17,7 @@ Flashing example for this keyboard:
 To reset the board into bootloader mode, do one of the following:
 
 * Hold the Reset switch mounted on the bottom side of the PCB while connecting the USB cable
-* Hold the Escape key while connecting the USB cable (also erases persistent settings)
```

Comment:
This refers to bootmagic lite, it shouldn't be removed

Reply:
Fair point, I tried the escape key method both on stock and on QMK and it didn't seem to work for me. I've pushed an update, can you review?

Reply:
Yea looks fine to me